tests/layer_tests/mo_python_api_tests/test_mo_convert_tf.py

```diff
@@ -940,14 +940,16 @@ def __call__(self, input):
         model = LayerModel()
         ov_model = convert_model(model)
 
+        # set model's output names will not change connected node names
+        # output can have additional names from connected input node
         ov_model.outputs[0].get_tensor().set_names({"name1"})
-        assert ov_model.outputs[0].names == {"name1"}
+        assert "name1" in ov_model.outputs[0].names
 
         ov_model.validate_nodes_and_infer_types()
-        assert ov_model.outputs[0].names == {"name1"}
+        assert "name1" in ov_model.outputs[0].names
 
         ov_model.outputs[0].get_tensor().set_names({"name2"})
-        assert ov_model.outputs[0].names == {"name2"}
+        assert "name2" in ov_model.outputs[0].names
```

Comment:
why now tensor names is changed? We should have the single tensor name instead of several